controller/src/main/java/io/pravega/controller/server/eventProcessor/requesthandlers/AbstractRequestProcessor.java

```diff
@@ -54,15 +55,16 @@
  * was set against its name.
  */
 @Slf4j
-public abstract class AbstractRequestProcessor<T extends ControllerEvent> extends SerializedRequestHandler<T> implements RequestProcessor {
+public abstract class AbstractRequestProcessor<T extends ControllerEvent> extends SerializedRequestHandler<T> implements StreamRequestProcessor {
     protected static final Predicate<Throwable> OPERATION_NOT_ALLOWED_PREDICATE = e -> Exceptions.unwrap(e) instanceof StoreException.OperationNotAllowedException;
 
-    protected final StreamMetadataStore streamMetadataStore;
+    @Getter
+    protected final StreamMetadataStore metadataStore;
```

Comment:
we dont need to add a getter. the protected variable is availble to derived classes. 

so revert these changes. And also revert all derived classes where you use the getter.

Reply:
done